plugins/protocolbuffers/cpp/v22.4/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1.4
+FROM debian:bullseye-20230502 AS build
+
+ARG TARGETARCH
+
+RUN apt-get update \
+ && apt-get install -y curl git cmake build-essential g++ unzip zip
+RUN arch=${TARGETARCH}; \
+    if [ "${arch}" = "amd64" ]; then arch="x86_64"; fi; \
+    curl -fsSL -o /usr/local/bin/bazel https://github.com/bazelbuild/bazel/releases/download/6.1.2/bazel-6.1.2-linux-${arch} \
```

Comment:
Bumped bazel in https://github.com/bufbuild/plugins/pull/516/commits/70943b965fb253a347a3344198699f0899248554 to latest minor https://github.com/bazelbuild/bazel/releases/tag/6.1.2